includes/Article.php

```diff
@@ -610,13 +610,24 @@ public function view() {
 					} elseif( !wfRunHooks( 'ArticleViewCustom', array( $this->mContent, $this->getTitle(), $wgOut ) ) ) {
 						# Allow extensions do their own custom view for certain pages
 						$outputDone = true;
-					} else {
+					} else {	
+						// wikia change begin
+						// VOLDEV-145
+						// used to suppress "redirect page" subtitle
+						// which is re-implemented in PageHeaderController::executeIndex
+						$appendSubTitle = !F::app()->checkSkin( ['oasis'] );
+						// end wikia change;
+
 						$text = $this->getContent();
 						$rt = Title::newFromRedirectArray( $text );
 						if ( $rt ) {
 							wfDebug( __METHOD__ . ": showing redirect=no page\n" );
 							# Viewing a redirect page (e.g. with parameter redirect=no)
-							$wgOut->addHTML( $this->viewRedirect( $rt ) );
+							// begin wikia change
+							// VOLDEV-145
+							$wgOut->addHTML( $this->viewRedirect( $rt, $appendSubtitle ) );
```

Comment:
There's an issue with links being added to breadcrumbs in inconsistent ways, due to sometimes manipulating what is output in core skins and adding it to a different breadcrumbs  and sometimes outputting it as is - compare the HTML and font-size on [redirect (action=delete)](http://runescape.wikia.com/wiki/KBD?action=delete) to [non-redirect (action=delete)](http://runescape.wikia.com/wiki/King_Black_Dragon?action=delete) with regards to `<- $page` link.

This is actually caused by a weird set of conditions when building the subtitle, see [here](https://github.com/Wikia/app/pull/9609/files?diff=unified#diff-518d114e986a0d2589af415d26592578R322) down to the end of the function. If it was me I'd unify the two subtitles, but I assumed there was a reason it was handled this way originally so I tried to make it more consistent instead.